free5gc-cp/README.md

```diff
@@ -0,0 +1,24 @@
+# free5gc-cp
+
+## Description
+sample description
```

Comment:
Please add a brief  description of the contents of this package

Reply:
@gvbalaji done, please review if it's sth you were thinking of